konami/dongle.md

```diff
@@ -0,0 +1,12 @@
+# Dongles
+
+Document about Konami's software and hardware license management using different kinds of dongles.
+
+## Types
+
+* Twinkle dongles (unnamed)
+* Roundplugs (miniDIN): serial one-wire (?), round "plug"
```

Comment:
Serial one-wire and Dallas iButton is basically the same thing (form factor aside), so it's better to unify the terminology here.

Reply:
Done.